src/shared/components/MarkDownEditor/MarkDownEditor.styles.ts

```diff
@@ -29,10 +29,6 @@ const commonStyles = `
     text-align: right;
   }
   
-  .md-editor-preview-wrapper img {
-    max-width: 50%;
```

Comment:
maybe better 100%, or it's by default?
because image width may be bigger than preview section width

Reply:
100% it's default for img